util/settings/settings.go

```diff
@@ -196,6 +204,21 @@ func (mgr *SettingsManager) getConfigMap() (*apiv1.ConfigMap, error) {
 	return argoCDCM, err
 }
 
+// Returns the ConfigMap with the given name from the cluster.
+// The ConfigMap must be labeled with "app.kubernetes.io/part-of: argocd" in
+// order to be retrievable.
+func (mgr *SettingsManager) GetNamedConfigMap(configMapName string) (*apiv1.ConfigMap, error) {
```

Comment:
minor -  all config map are named - aren't they - how about `GetConfigMapByName`?

Reply:
Much better, clear & concise. Will rename it, thanks!